instances/atlas/parcellationEntity/SWMA/SWMA_CMF-PrC.jsonld

```diff
@@ -0,0 +1,30 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/SWMA_CMF-PrC",
+  "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+  "hasParent": [
+    {
+      "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/SWMA_superficialFibreBundles"
+    }
+  ],
+  "hasVersion": [
+    {
+      "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/SWMA_2018_lh_CMF-PrC_0"
+    },
+    {
+      "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/SWMA_2018_lh_CMF-PrC_1"
```

Comment:
this is not a version of CMF-PrC. This is a substructure. 

Relationship should be:
+ CMF-PrC (as PE)
   + CMF-PrC_0 (as PE)
      + lh_CMF-PrC_0 (as PEV)
      + rh_CMF-PrC_0 (as PEV)
   + CMF-PrC_1 (as PE)
      + lh_CMF-PrC_1 (as PEV)
      + rh_CMF-PrC_1 (as PEV)

Reply:
I'm sorry I overlooked that issue before.

Reply:
@lzehl , question, in your example,

- would `lh_CMF-PrC_1 (as PEV)` : `hasParent` field be `CMF-PrC_1 (as PE)` ? (This would directly contradict the first review comment: `For the PEVs: They should not have a parent.` -> which is fine, I just would like to confirm)
- or would `CMF-PrC_1 (as PE)` : `hasVersion` field be `lh_CMF-PrC_1 (as PEV)` ?
- or both of above?